build.gradle.kts

```diff
@@ -11,23 +11,21 @@ version = "3.1.1"
 val targetJavaVersion = 8
 
 // Dependencies
-
 repositories {
     mavenCentral()
-
     maven("https://hub.spigotmc.org/nexus/content/groups/public/")
     maven("https://repo.dmulloy2.net/repository/public/")
 }
 
 dependencies {
-    compileOnly("org.bukkit:bukkit:1.9.4-R0.1-SNAPSHOT")
-    compileOnly("io.netty:netty-all:4.1.79.Final")
+    compileOnly("org.spigotmc:spigot-api:1.9.4-R0.1-SNAPSHOT")
```

Comment:
I kept it at 1.9.4, since most of the development I do privately is in the latest version and I didn't want to accidentally use an API that's not available in that version (I figured since a lot of APIs are backward compatible that case is a lot more likely than accidentally using one that's been removed). Other side of that medal is of course that I don't notice deprecation warnings or removed API.

Ideally I should just set up GitHub Actions with a proper test setup. I read about some nice testing libraries for Minecraft, but integration tests are always *meh* in this context. Until that's happened I'd prefer to leave it at the old version. 

Alternatively, we could make a separate gradle subproject for each version adapter (which is how most projects handle it). Not sure if that's getting too complicated for a simple plugin like this though.

Reply:
> Alternatively, we could make a separate gradle subproject for each version adapter (which is how most projects handle it). Not sure if that's getting too complicated for a simple plugin like this though.

I also thought about this but didn't implemented it because i didn't know if this is at all wanted, but if wanted I can try to implement this in Gradle. Should I try it?

Reply:
@RodneyMKay Should I use CraftBukkit via `mavenLocal()` Gradle repository directly or stay at the use of Reflection?

Reply:
Nah, I think leaving the version as is is good enough for now. Thanks for the offer though ^^